src/Propel/Generator/Util/QuickBuilder.php

```diff
@@ -235,9 +274,9 @@ public function build($dsn = null, $user = null, $pass = null, $adapter = null,
     /**
      * @return \Propel\Generator\Model\Database|null
      */
-    public function getDatabase()
+    public function getDatabase(): ?Database
     {
-        if ($this->database === null) {
+        if (!isset($this->database)) {
```

Comment:
What's the reason behind this change? !isset is slower than === null

Reply:
Fixed.